### PR TITLE
Support multiple control and test clusters in Verifier

### PR DESCRIPTION
**Pull request by caithagoras**
```
== RELEASE NOTES ==

Verifier Changes
* Add support to allow multiple control and test clusters.
```

### CODE REVIEW

File: presto-verifier/src/main/java/com/facebook/presto/verifier/prestoaction/PrestoActionConfig.java [hunk 1]
```diff
@@ -33,23 +37,28 @@
...
 
-    @Config("host")
-    public PrestoActionConfig setHost(String host)
+    @Config("hosts")
+    @ConfigDescription("Comma-separated list of the control cluster hostnames or IP addresses.")
+    public PrestoActionConfig setHosts(String hosts)
     {
-        this.host = host;
+        if (hosts != null) {
+            this.hosts = HOST_SPLITTER.splitToList(hosts);
```
Comment (sujay-jain) [inline presto-verifier/src/main/java/com/facebook/presto/verifier/prestoaction/PrestoActionConfig.java:60]: Should we use ```ImmutableList.copyOf()```?

**Review by sujay-jain — APPROVED** [review]
LGTM % comment

**Comment by caithagoras** [comment]
@mbasmanova comments addrssed

**Comment by mbasmanova** [comment]
Perfect. Thank you for reviewing @sujay-jain . @caithagoras This is good to merge once the tests are green.

**Pull request merged**